mathjax3-ts/output/chtml/Wrapper.ts

```diff
@@ -491,7 +500,7 @@ export class CHTMLWrapper extends AbstractWrapper<MmlNode, CHTMLWrapper> {
         if (!this.styles) return;
         const styles = this.styles.cssText;
         if (styles) {
-            this.chtml.style.cssText = styles;
+            this.adaptor.setAttribute(this.chtml, 'style', styles);
```

Comment:
Why is that not setStyle? Or is this supposed to overwrite the existing style.

Reply:
The `setStyle()` method sets a single style by passing the style selector and its new value.  But here we are setting the complete style attribute (e.g., `styles` might be `'border: none; padding: 3px 5px; font-family: Times, serif'` or something like that).  Setting the `'style'` attribute clears the existing styles and replaces them with the new ones (just as setting `style.cssText` did before).  So this is the correct usage.